packages/s/spirv-headers/xmake.lua

```diff
@@ -11,6 +11,7 @@ package("spirv-headers")
     add_urls("https://github.com/KhronosGroup/SPIRV-Headers/archive/$(version).tar.gz", {version = function (version) return map[tostring(version)] end})
     add_versions("1.5.4+2", "df2ad2520be4d95a479fa248921065885bbf435a658349a7fc164ad7b26b68c6")
     add_versions("1.5.5", "9eb56548460fd8850250ebf78071528fb66c2a5db2ef535edc1d493b2581ec66")
+    add_versions("sdk-1.2.198.0", "3301a23aca0434336a643e433dcacacdd60000ab3dd35dc0078a297c06124a12")
```

Comment:
we only support semver https://semver.org/. 

try `1.2.198+0-sdk` and add map item

Reply:
and we maybe need add this sdk version to vulkan validator package